src/SCREENS.ts

```diff
@@ -219,6 +219,7 @@ const SCREENS = {
         TAGS: 'Workspace_Tags',
         TAGS_SETTINGS: 'Tags_Settings',
         TAGS_EDIT: 'Tags_Edit',
+        TAXES: 'Workspace_Taxes',
```

Comment:
We should wrap it into a section ([design doc](https://docs.google.com/document/d/12FZycslrIE95QoKM8DZ37H_wLbH9w1jt3uA-a6BCqtc/edit?disco=AAABGiXKfFc)).

Reply:
Can't we do something like this?
```suggestion
        TAXES: {
           ROOT: 'Workspace_Taxes',
           SETTINGS: 'Workspace_Taxes_Settings',
           NEW: 'Workspace_Taxes_New',
           // ...
        },
```

Reply:
We can't make this change without modifying WorkspaceInitialPage.tsx, which could involve substantial updates.